app/main.py

```diff
@@ -27,28 +28,42 @@ def load_data(file_path: str) -> list[dict]:
 
 
 def video_with_highest_views(videos) -> str:
-    pass
+    videos.sort(key=lambda vide0: vide0['views'], reverse=True)
+    return videos[0]['title']
 
 
 # ration = likes / views
 def average_likes_to_views_ratio(videos: list[dict]) -> float:
-    pass
+    return sum([vide0["likes"] / vide0["views"] for vide0 in videos if vide0["views"] != 0]) / len([vide0["likes"] / vide0["views"] for vide0 in videos if vide0["views"] != 0])
```

Comment:
maybe it would be better to make a variable 
`some_var = [vide0["likes"] / vide0["views"] for vide0 in videos if vide0["views"] != 0]`
and then use it, because know you have code duplications